yt/frontends/_skeleton/__init__.py

```diff
@@ -1,14 +1,14 @@
 """
-API for yt.frontends.skeleton
+API for yt.frontends._skeleton
 
 
 
 """
 
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
 # Copyright (c) 2013, yt Development Team.
 #
 # Distributed under the terms of the Modified BSD License.
 #
 # The full license is in the file COPYING.txt, distributed with this software.
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
```

Comment:
I can do this--I'll probably rebase then.
I was inspired by `pep8` spitting out an error (specifically, [E265](http://pep8.readthedocs.io/en/latest/intro.html#error-codes)).
It really looks like this extra space should be there, because then the last `-` character becomes the 79th, which would be perfect or, say, expected--as opposed to an exotic 78th...
But I understand it's the Copyright header so it's touchy.